dotCMS/src/integration-test/java/com/dotmarketing/business/web/HostWebAPIImplIntegrationTest.java

```diff
@@ -0,0 +1,721 @@
+package com.dotmarketing.business.web;
+
+import com.dotcms.IntegrationTestBase;
+import com.dotcms.datagen.RoleDataGen;
+import com.dotcms.datagen.SiteDataGen;
+import com.dotcms.datagen.UserDataGen;
+import com.dotcms.util.CollectionsUtils;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.beans.Permission;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.PermissionAPI;
+import com.dotmarketing.business.Role;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import static junit.framework.TestCase.assertEquals;
+import static org.mockito.Mockito.*;
+
+import com.dotmarketing.util.WebKeys;
+import com.liferay.portal.model.User;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+
+public class HostWebAPIImplIntegrationTest extends IntegrationTestBase {
+
+    public static final String HOST_ID_PARAMETER_NAME = "host_id";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "host_id" appears 15 times in this file; the first occurrence is on line 29](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4812018)